modules/core/02-client/keeper/events.go

```diff
@@ -0,0 +1,106 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/cosmos/ibc-go/v2/modules/core/02-client/types"
+	"github.com/cosmos/ibc-go/v2/modules/core/exported"
+)
+
+// EmitCreateClientEvent emits a create client event
+func EmitCreateClientEvent(ctx sdk.Context, clientID string, clientState exported.ClientState) {
```

Comment:
I see that all these other `Emit...` functions were added as part of [#783](https://github.com/cosmos/ibc-go/pull/783), but not back ported to any of the v2 release lines, so if we add then in this PR they are actually not used in the code. Should we remove then so that we also keep closer to the spirit of the PR that is being back ported here?

We have this problem in the other v2 back ports as well...